artifacts/deploy/karmada-apiserver.yaml

```diff
@@ -43,7 +43,6 @@ spec:
             - --etcd-keyfile=/etc/kubernetes/pki/karmada.key
             - --etcd-servers=https://etcd-client.karmada-system.svc.cluster.local:2379
             - --bind-address=0.0.0.0
-            - --insecure-port=0
```

Comment:
This flag has been removed from v1.24?

Reply:
Yes, it is.

![image](https://user-images.githubusercontent.com/78244870/177928641-f0021247-1753-4d24-ae02-612458f8486f.png)

https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#changes-by-kind-2

Reply:
Should we also modify the `karmada init` stage by removing the flags ?